### PR TITLE
fix: Add config for max task runners payload size (no-changelog)

### PR DESCRIPTION
**Pull request by valya**
## Summary

Workflow payloads can end up being quite large, this increases the default max sent over the Task Runner websockets, and adds an option to configure it.

## Review / Merge checklist

- [ ] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/@n8n/config/src/configs/runners.config.ts [hunk 1]
```diff
@@ -32,6 +32,10 @@ export class TaskRunnersConfig {
 	@Env('N8N_RUNNERS_SERVER_LISTEN_ADDRESS')
 	listenAddress: string = '127.0.0.1';
 
+	/** Maximum size of a payload sent to the runner in bytes */
```
Comment (tomi) [inline packages/@n8n/config/src/configs/runners.config.ts:35]:
```suggestion
	/** Maximum size of a payload sent to the runner in bytes. Default is 1G */
```

**Review by tomi — DISMISSED** [review]
Besides one addition to the docs LGTM 🚀

**Review by tomi — APPROVED** [review]
🚀

**Comment by janober** [comment]
Got released with [`n8n@1.65.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.65.0)

**Pull request merged**